src/main/java/Applicaiton.java

```diff
@@ -0,0 +1,214 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import enumeration.Commands;
+
+import static enumeration.Commands.*;
+
+import enumeration.Genre;
+
+import static enumeration.Genre.*;
+
+import models.Song;
+import models.Singer;
+
+/**
+ * The Application class is the main class in the application.
+ */
+public class Applicaiton {
+    private AudioPlayer audioPlayer = new AudioPlayer(this);
+    private Commands currentCommand = PLAY;
+    private Commands previousCommand;
+    private BufferedReader inputReader = new BufferedReader(new InputStreamReader(System.in));
+
+    public static void main(String args[]) throws InterruptedException, IOException {
+
+        Applicaiton app = new Applicaiton();
+        List<Song> songList = new ArrayList<>();
+        songList.add(new Song("High way to hell", new Singer("John"), ROCK, 100));
+        songList.add(new Song("Thunderstruck", new Singer("Alex"), ROCK, 100));
+        songList.add(new Song("High voltage", new Singer("Silvia"), ROCK, 100));
+        app.audioPlayer.addListOfSongs(songList);
```

Comment:
Instead of creating one more `List`, you can create a method `addSong(Song song)`